### PR TITLE
Documentation for Artipie scripting support

### PR DESCRIPTION
**Pull request by ChGen**
Documentation for Artipie scripting support. Part of https://github.com/artipie/artipie/issues/1277

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review.

**Review by olenagerasimova — COMMENTED** [review]
thanks, looks good. Though, I'd suggest to add simple script example (one, in any language) with access to artipie objects.

**Comment by ChGen** [comment]
@dgarus , please review. I added snippet example.

File: .wiki/Configuration-Scripting.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| Ruby               | .rb            |
+
+### Accessing Artipie objects
+
+Some Artipie objects could be accessed from the scripts. Such objects have names starting with underscore `_`.
+Table with available objects is given below.
+
+| Object name       | Artipie type                                |
+|-------------------|---------------------------------------------|
+| `_settings`       | com.artipie.settings.Settings               |
```
Comment (dgarus) [inline .wiki/Configuration-Scripting.md:36]: It could be great to add links to the java docs or, at least, to files in the master branch.

**Review by dgarus — CHANGES_REQUESTED** [review]
Thanks!
There are a few minor comments.

**Review by dgarus — APPROVED** [review]
lgtm

**Pull request merged**